lib/fenrir_view/presenter.rb

```diff
@@ -5,10 +5,12 @@ class Presenter
 
     attr_reader :variant, :slug, :properties
 
-    def initialize(variant, slug, properties = {})
+    def initialize(variant, slug, properties = {}, validate = true)
```

Comment:
I think I'd make this `validate: true` as a named parameter. 

Makes

```
@component_facade ||= FenrirView::Presenter.component_for(variant, name, {}, validate: false)
```

A bit more self documenting.

Reply:
Ah yeah will make this change, I didn't initially to avoid mixing named and unnamed, but this makes more sense.